DataStructures.Tests/HashTables/HashTableTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using DataStructures.HashTables;
+using System.Net.Http.Headers;
+
+namespace DataStructures.Tests.HashTables
+{
+    public class HashTableTests
+    {
+        [Fact]
+        public void ReturnsHash()
+        {
+            int x = HashTable<string>.GetHash("Dog");
+            var d = 'D';
+            var o = 'o';
+            var g = 'g';
+            Assert.Equal(( (int)d + (int)o + (int)g) * 269, x);
+        }
+        [Fact]
+        public void HashListDoesHashListThings()
+        {
+            var list = new HashList<string>(1);
+
+            list.Insert(77, "Chicken");
+            list.Insert(12, "Taco");
+
+            Assert.NotNull(list.Head);
+            Assert.True(list.Includes(12));
+            Assert.Equal("Chicken", list.KeyValue(77));
+        }
+        [Fact]
+        public void HashTableStuffWorksQuickTests()
+        {
+            var table = new HashTable<string>(100);
+            table.Add("Taco", "Beef");
+            Assert.Equal("Beef", table.Get("Taco"));
+        }
+
+        [Fact]
+        public void TryFindTest()
+        {
+            var table = new HashTable<string>(100);
+            table.Add("Taco", "Beef");
+            string x;
+            Assert.True(table.TryFind("Taco", out x));
```

Comment:
```suggestion
            Assert.True(table.TryFind("Taco", out string x));
```


Also, noticed you don't have a test for `TryFind()` returning `false`!